### PR TITLE
Adding config files for convnext

### PR DESCRIPTION
**Pull request by soma2000-lang**
# What does this PR do?
Adding config files for convnext
Based on issue: https://github.com/huggingface/transformers/issues/19487

@ydshieh could you please check it?
Thanks :)

### CODE REVIEW

**Comment by soma2000-lang** [comment]
Ok

**Comment by soma2000-lang** [comment]
image
@ydshieh After running make style I am getting the above.I am unable to understand what cold have been the problem

**Comment by ydshieh** [comment]
OK. First thing to resolve: your PR also changes the following file (and also in a strange way)
```
src/transformers/models/clip/configuration_clip.py
```

The goal of this PR is to work on convnext. We should not change the above file. Could you try to make this PR clean by not doing changes to CLIP config filei 🙏 ?

**Comment by soma2000-lang** [comment]
@ydshieh My last pr was about adding config files for the src/transformers/models/clip/configuration_clip.py thats  why changes were also reflected in this pr .Thats why I deleted the src/transformers/models/clip/configuration_clip.py file.

**Comment by ydshieh** [comment]
You can't just delete that file, as it means the file will be deleted when the PR is merged to the library.
You should instead to revert that file to the original state.

File: src/transformers/models/convnext/configuration_convnext.py [hunk 1]
```diff
@@ -67,11 +67,11 @@ class ConvNextConfig(PretrainedConfig):
 
     Example:
     ```python
-    >>> from transformers import ConvNextModel, ConvNextConfig
+    >>> from transformers import ConvNextConfig, ConvNextModel
 
     >>> # Initializing a ConvNext convnext-tiny-224 style configuration
     >>> configuration = ConvNextConfig()
-    >>> # Initializing a model from the convnext-tiny-224 style configuration
+    >>> # Initializing a model (with random weights) from the convnext-tiny-224 style configuration
```
Comment (ydshieh) [inline src/transformers/models/convnext/configuration_convnext.py:74]: Let's add an empty line before this line

Comment (ydshieh) [reply src/transformers/models/convnext/configuration_convnext.py:74]: Also an empty line between `>>> model = ConvNextModel(configuration)` and `>>> # Accessing the model configuration`

File: utils/documentation_tests.txt [hunk 2]
```diff
@@ -151,4 +151,5 @@ src/transformers/models/xlnet/configuration_xlnet.py
 src/transformers/models/yolos/configuration_yolos.py
 src/transformers/models/yolos/modeling_yolos.py
 src/transformers/models/x_clip/modeling_x_clip.py
-src/transformers/models/yoso/configuration_yoso.py
+src/transformers/models/yoso/configuration_yoso.pyt
+src/transformers/models/convnext/configuration_convnext.py
```
Comment (ydshieh) [inline utils/documentation_tests.txt:155]: Let's put this line in a correct order inside this file 🙏

**Review by ydshieh — COMMENTED** [review]
Thank you @soma2000-lang . We are almost done. Just 3 tiny changes to make 🙏 🚀

File: utils/documentation_tests.txt [hunk 3]
```diff
@@ -151,4 +152,4 @@ src/transformers/models/xlnet/configuration_xlnet.py
 src/transformers/models/yolos/configuration_yolos.py
 src/transformers/models/yolos/modeling_yolos.py
 src/transformers/models/x_clip/modeling_x_clip.py
-src/transformers/models/yoso/configuration_yoso.py
+src/transformers/models/yoso/configuration_yoso.pyt
```
Comment (ydshieh) [inline utils/documentation_tests.txt:155]: got an extra `t` here. Let's remove it and we are good to merge :-)

File: utils/documentation_tests.txt [hunk 4]
```diff
@@ -79,7 +80,7 @@ src/transformers/models/longt5/modeling_longt5.py
 src/transformers/models/marian/modeling_marian.py
 src/transformers/models/markuplm/modeling_markuplm.py
 src/transformers/models/mbart/modeling_mbart.py
-src/transformers/models/megatron_bert/configuration_megatron_bert.py 
+src/transformers/models/megatron_bert/configuration_megatron_bert.py
```
Comment (ydshieh) [inline utils/documentation_tests.txt:83]: nice catch!

**Review by ydshieh — APPROVED** [review]

**Pull request merged**